owncloudApp/src/main/res/values/strings.xml

```diff
@@ -748,6 +748,8 @@
     <string name="release_notes_4_3_0_subtitle_improvements_remove_dialog">Added a custom dialog when the file that is going to be deleted has thumbnail and added the number of files that are going to be deleted in a multiple selection</string>
     <string name="release_notes_4_3_0_title_password_generator">Password generator for public links</string>
     <string name="release_notes_4_3_0_subtitle_password_generator">Passwords for public links can be now generated via a new generator in Infinite Scale accounts</string>
+    <string name="release_notes_4_3_0_title_display_name_truncation">Resolved the bug where long display names were truncated incorrectly</string>
+    <string name="release_notes_4_3_0_subtitle_display_name_truncation">Display names are now properly truncated in the middle with ellipsis (...) to maintain readability.</string>
```

Comment:
Here, by convention, we don't add a final dot to the sentences, so better to remove it 😄 
```suggestion
    <string name="release_notes_4_3_0_subtitle_display_name_truncation">Display names are now properly truncated in the middle with ellipsis (...) to maintain readability</string>
```